dashboard/test/models/pd/workshop_test.rb

```diff
@@ -363,13 +363,13 @@ class Pd::WorkshopTest < ActiveSupport::TestCase
   end
 
   test 'plp' do
-    assert_nil @workshop.professional_learning_partner
+    assert_nil @workshop.regional_partner
 
     # Now create a plp associated with the organizer
```

Comment:
Now create an RP...

Consider searching GitHub for "PLP" and "professional learning partner", updating all occurrences as appropriate.